### DISCUSSION TITLE
Livewire Wiretap

### DISCUSSION THREAD

**Discussion by aptcreativedesign** [livewire/livewire#6682] [4. General]
Hi All,

I don't know how many of you are actively using Livewire v3 for your projects at the moment.

The fact that browser Livewire Dev tools is no longer compatible with v3 is beginning to effect workflow and debugging for me and my team.

I saw Caleb's Laracon presentation this year and he announced Wiretap, which looks awesome! 

Does anyone (@calebporzio) have any idea on timeframe for the release of Wiretap?

**Comment by aptcreativedesign** [comment]
Last month Chrome Livewire dev tools started working with V3. This was great news for our dev team.

Not sure if any of the other browser tools are now working but for those who don't use it I highly recommend the chrome plugin [dev tools](https://chrome.google.com/webstore/detail/livewire-devtools/dnociedgpnpfnbkafoiilldfhpcjmikd)

Can't wait for Caleb to be able to spend the time needed on wiretap but totally understand his need to make money first

**Comment by malzariey** [comment]
Cant wait to hear any news as well!

**Comment by marcorieser** [comment]
He mentioned in the Laravel Worldwide Meetup that it's currently not on his roadmap: 
https://www.youtube.com/live/5cdYaQbzgHE?si=mDIjNufmDVJVwvV4&t=4498

**Comment by hardevine** [comment]
anything about wiretap ? 😢 
anyone tried debugbar for livewire debugging ?
